src/ast/expr/LambdaExpr.php

```diff
@@ -52,18 +52,21 @@ public function format(Parser $parser)
                 break;
             case 1:
                 if ($this->has_brackets) {
-                    $source .= '['
-                             . ($this->parameters[0]->is_reference ? '*' : '')
-                             . $this->parameters[0]->name
-                             . ']';
+                    $source .= '[' . $this->parameters[0]->name . ']';
                 } else {
                     $source .= $this->parameters[0]->name;
                 }
                 break;
             default:
                 $source .= '[';
                 $source .= implode(', ', array_map(function ($param) {
-                    return ($param->is_reference ? '*' : '') . $param->name;
+                    $parameter = $param->name;
+
+                    if (isset($param->type)) {
```

Comment:
using `isset` here doesn't go against our current member checking? `NULL !== $param->type`
Or is that another reason that I did not see? :) just wondering ^^